site/src/docs/components/cookie-consent/api.mdx

```diff
@@ -0,0 +1,135 @@
+| -------------------------------------------------------------------------------------------------------------------------------- | -------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `siteName`                                                                                                                       | **Yes**  | Cookie consent main heading.                                                                                                                                                                                                                                                                                           |
+| `currentLanguage`                                                                                                                | **Yes**  | Currently selected language. Used for picking translations from `content.json`. Is also set to `content.language.current`. Supported values are `fi`, `sv`, and `en`.                                                                                                                                                   |
+| `requiredCookies`                                                                                                                | No       | Required cookies can have partial data or all data described in the table below. Missing texts are auto-filled. See examples below this table.                                                                                                                                                                         |
+| `optionalCookies`                                                                                                                | No       | Optional cookies can have partial data or all data described in the table below. Missing texts are auto-filled. See examples below this table.                                                                                                                                                                          |
+| `requiredConsents.groups`, `optionalConsents.groups`                                                                             | No       | List of cookie groups. All cookies must belong to a group. Groups with property `commonGroup` will be auto-filled from `content.json`.                                                                                                                                                                                  |
```

Comment:
Should this be `requiredCookies.groups`? I was unable to find the property `requiredConsents` from the `contentSource` property.

Reply:
Fixed